project_stage_warning/models/project_task.py

```diff
@@ -0,0 +1,24 @@
+# -- coding: utf-8 --
+from odoo import models, fields, api, _
+from odoo.exceptions import UserError
+
+
+class ProjectTask(models.Model):
+    _inherit = "project.task"
+
+    def write(self, values):
+        if values.get('stage_id'):
+            new_stage = self.env['project.task.type'].browse(values.get('stage_id'))
+            end_sequence = new_stage.sequence
+            for task in self:
+                start_sequence = task.stage_id.sequence
+                skipped_stage = self.env['project.task.type'].search([
```

Comment:
`search_count` for perf?

Reply:
With `search_count` we don't have the name of the stage to show in the warning.
We might use `search_read` instead?